### ISSUE TITLE
Add ability to cache longer than 30 days

### ISSUE THREAD

**Issue by harrydhillon43** [sixhours-team/memcached-spring-boot#152]
We wanted to cache certain items for longer than 30 days.  In our case, requirements were cache for 40 days.

Would it be possible to remove this validation or bump the limit to 60?

```
if (expiration == null || expiration.toDays() > 30) {
     throw new IllegalStateException("Invalid expiration. It should not be null or greater than 30 days.");
}
```

**Comment by igorbolic** [comment]
@harrydhillon43 The support for caching more than 30 days is available with **2.5.0-SNAPSHOT** version. You can give it a try and check if it works for you, until we release the final version.

This version is available at the snapshot Maven repository: https://oss.sonatype.org/content/repositories/snapshots

**Comment by igorbolic** [comment]
Supported with release version **2.5.0**
